pyxlma/lmalib/flash/properties.py

```diff
@@ -182,6 +225,15 @@ def flash_stats(ds):
                                                             df['event_y'],
                                                             df['event_z']))
 
+    #Compute flash discharge energy using parallel plate capacitor: (probably
+    #a better way to get the areas without having to have an embedded function
+    #call?)
+    event_energy = fl_gb.apply(lambda df: event_discharge_energy(df['event_z'],
+                                                                 event_hull_area(df['event_x'],
```

Comment:
The areas are already calculated on an earlier line, so pass the area array directly instead of recalculating.

Reply:
I tried, but it didn't work. When re-calculating the areas the flash-sorting runs fast. But if passing in 'event_area' or 'event_area.values', it hangs or throws an error.